spring-boot-project/spring-boot-docs/src/main/asciidoc/deployment.adoc

```diff
@@ -143,9 +146,10 @@ Spring's `Environment` abstraction:
 	}
 ----
 
-All Cloud Foundry properties are prefixed with `vcap`. You can use vcap properties to
+All Cloud Foundry properties are prefixed with `vcap`. You can use `vcap` properties to
 access application information (such as the public URL of the application) and service
-information (such as database credentials). See `CloudFoundryVcapEnvironmentPostProcessor`
+information (such as database credentials). See
+https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/cloud/CloudFoundryVcapEnvironmentPostProcessor.html['`CloudFoundryVcapEnvironmentPostProcessor`']
```

Comment:
This should use the `{dc-spring-boot}` attribute defined in index.adoc.

Reply:
Yup. Thanks.